spark/agent/src/main/scala/za/co/absa/spline/harvester/conf/DefaultSplineConfigurer.scala

```diff
@@ -65,7 +70,17 @@ class DefaultSplineConfigurer(configuration: Configuration, sparkSession: SparkS
     }
   }
 
-  override lazy val lineageDispatcher = HttpLineageDispatcher(configuration)
+  override lazy val lineageDispatcher: LineageDispatcher = {
+    configuration.getString(LINEAGE_DISPATCHER_CLASS, LINEAGE_DISPATCHER_CLASS_DEFAULT) match {
+      case LINEAGE_DISPATCHER_CLASS_DEFAULT => HttpLineageDispatcher(configuration)
+      case className =>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Lower case pattern matching.](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4517665)

Reply:
Please address this one. LGTM otherwise.